vehicle/mkv/software/dashboard/dashboard_config.h

```diff
@@ -0,0 +1,37 @@
+#include "libs/adc/api.h"
+#include "libs/can/api.h"
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+#include "libs/timer/api.h"
+#include <avr/interrupt.h>
+#include <avr/io.h>
+
+#define RTD_BUZZ_TIME 4000 // milliseconds
+
+// Pin definitions
+gpio_t IMD_LED = PB3;
+gpio_t START_LED = PB4;
+gpio_t AMS_LED = PD0;
```

Comment:
Can you rename to `BMS_LED`? The rules term is `AMS` but we use BMS internally so let's stick with that.